src/main/java/vbn/state/constraints/BinaryConstraint.java

```diff
@@ -18,12 +18,14 @@ public BinaryConstraint(@NonNull Symbol left, @NonNull BinaryOperand op, @NonNul
         this.left = left;
         this.op = op;
         this.right = right;
+        this.negated = false;
```

Comment:
```suggestion
```

If you could remove this...

Reply:
Fixed in [c20af33](https://github.com/CMPT-479/CMPT-479-Project/pull/32/commits/c20af330db3eb5857b775153c9499980e5da16a4) and [cefec5c](https://github.com/CMPT-479/CMPT-479-Project/pull/32/commits/cefec5c6ca5e3350d7c07a128c30b406a65d884a)